specification/cognitiveservices/Language.Conversations/main.tsp

```diff
@@ -1,12 +1,13 @@
-import "@typespec/rest";
 import "@typespec/http";
 import "@typespec/versioning";
+import "@typespec/openapi";
 import "@azure-tools/typespec-azure-core";
 import "@azure-tools/typespec-client-generator-core";
+import "@azure-tools/typespec-autorest";
 import "./common.tsp";
 
 using TypeSpec.Http;
-using TypeSpec.Rest;
```

Comment:
Why remove this and then fully qualify it throughout the spec? I mean, it's not wrong, but seems like extra work to maintain - especially for discoverability for people just ramping up on TypeSpec.